net-p2p/tremc/tremc-0.9.2.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7,8} )
+PYTHON_REQ_USE="ncurses"
+inherit bash-completion-r1 python-single-r1
+
+DESCRIPTION="Ncurses interface for the Transmission BitTorrent client"
+HOMEPAGE="https://github.com/tremc/tremc"
+SRC_URI="https://github.com/tremc/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="geoip"
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+RDEPEND="${PYTHON_DEPS}
+	$(python_gen_cond_dep '
+		geoip? ( dev-python/geoip-python[${PYTHON_USEDEP}] )
+	')
```

Comment:
I don't like this optional runtime dependency. You should move it to `optfeature` using *eutils.eclass*. Note that this syntax does work in `optfeature` but makes the output ugly, hence I'd suggest just putting `dev-python/geoip-python` in there. If user is running with ... non-messed up configuration, system default and PYTHON_SINGLE_TARGET should be the same python interpreter. And if they are not, I doubt this is the first package to introduce breakage...

There also seem to be other optional dependencies, 
```
# optional features provided by non-standard modules                                                
features = {'dns': False, 'geoip': False, 'ipy': False, 'clipboard': False}                         
try:   import adns; features['dns'] = True     # resolve IP to host name                            
except ImportError: features['dns'] = False                                                         
                                                                                                    
try:   import GeoIP; features['geoip'] = True  # show country peer seems to be in                   
except ImportError:  features['geoip'] = False                                                      
                                                                                                    
try:   import IPy;  features['ipy'] = True  # extract ipv4 from ipv6 addresses                      
except ImportError: features['ipy'] = False         
```
Please add them to `optfeature` as well.

Reply:
Touched up. Doesn't appear we have an `adns` in Portage so I didn't include that.